commune_app/all_models/chores.py

```diff
@@ -4,19 +4,19 @@
 
 class Chore(models.Model):
     title = models.CharField(max_length=50)
+    description = models.TextField(max_length=300)
     date = models.DateTimeField(auto_now_add=True)
     budget = models.DecimalField(max_digits=7, decimal_places=2, null=True)
-    assign = models.ForeignKey(User, on_delete=models.CASCADE)
-    description = models.TextField(max_length=300)
+    assign_to = models.ForeignKey(User, on_delete=models.CASCADE)
     passed = models.BooleanField()
+    status = models.BooleanField()
```

Comment:
I thought about an enum with a few options like: NEW, IN_PROGRESS, COMPLETED
but maybe you can do it with "completed" boolean variable(instead of status)
feel free to decide which one you want to implement

Reply:
changed it to completed field and we have the passed field aswell..